stix_shifter/stix_translation/src/modules/bigfix/bigfix_query_constructor.py

```diff
@@ -69,6 +48,7 @@ def _parse_expression(self, expression, qualifier=None):
             stix_object, stix_remainings = expression.object_path.split(':')
             value = self._escape_value(expression.value)
             comparator = self.comparator_lookup[expression.comparator]
+            final_value = comparator + ' "' + value + '"'
             stix_property = ""
             stix_key = ""
             self.query_type = stix_object
```

Comment:
single hash query can also be interpreted as file hash query not just process. if file hash passed into stix pattern, it can also be someone looking for files with that hash. not sure how are going to differentiate that?

Reply:
Well if we just have a file hash we do not have any directories so I think processes works better